scripts/output.py

```diff
@@ -15,14 +16,23 @@ def read_input():
             # print(extracted_values)
             extracted_values.insert(0, extracted_values[3])
             extracted_values.pop()
+            extracted_values.insert(1, extracted_values[2])
+            del extracted_values[3]
```

Comment:
Hmm, low priority but could we just use a dictionary for all of this? Hopefully less error prone and then we should never have issues with column alignment like we have previously